jekyll/_cci2/config-editor.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: classic-docs
+title: "Using the CircleCI In-app Configuration Editor"
+description: "Docs page on In-app configuration editor use and features"
+version:
+- Cloud
+- Server v2.x
```

Comment:
Is this feature available on server 2.x? If not, we'll want to remove this tag.

Reply:
@teesloane double checking